### PR TITLE
HTTP/2 code sharing shell sync-script

### PR DESCRIPTION
**Pull request by gfoidl**
ASP.NET Core part of https://github.com/dotnet/corefx/pull/42425

### CODE REVIEW

File: src/Shared/Http2/CopyToCoreFx.cmd [hunk 1]
```diff
@@ -11,4 +11,4 @@ IF [%remote_repo%] == [] (
 echo COREFX_REPO: %remote_repo%
 
 robocopy . %remote_repo%\src\Common\src\System\Net\Http\Http2 /MIR
-robocopy .\..\test\Shared.Tests\Http2 %remote_repo%\src\Common\tests\Tests\System\Net\Http2 /MIR
+robocopy .\..\test\Shared.Tests\Http2 %remote_repo%\src\Common\tests\Tests\System\Net\Http2 /MIR
```
Comment (gfoidl) [inline src/Shared/Http2/CopyToCoreFx.cmd]: This change come from `./CopyToAspNetCore.sh`-syncing with corefx.

**Comment by gfoidl** [comment]
I won't push further updates, as long the corefx-PR isn't done to avoid (unnecesary) CI-builds.

**Comment by Tratcher** [comment]
Does this need any updates now that the corefx version has merged?

**Comment by gfoidl** [comment]
@Tratcher I just pushed one last commit, now the versions are the same, and this PR is ready to merge.

**Review by Tratcher — APPROVED** [review]

**Comment by Tratcher** [comment]
Thanks again

**Pull request merged**